### ISSUE TITLE
sitewide 508-defect-2 [RESPONSIVE]: Expanded profile menu MUST be navigable on zoom

### ISSUE THREAD

**Issue by joshkimux** [department-of-veterans-affairs/va.gov-team#19784]
# [508-defect-2](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md#508-defect-2)

## Feedback framework

- **❗️ Must** for if the feedback must be applied 
- **⚠️ Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Definition of done

1. Review and acknowledge feedback.
1. Fix and/or document decisions made.
1. Accessibility specialist will close ticket after reviewing documented decisions / validating fix.

## Point of Contact

**VFS Point of Contact:** _Josh_

## User Story or Problem Statement

As a user with low vision, I expect to be able to navigate VA.gov using 400% zoom.

## Details

Users who use magnification and users who use smaller _vertical_ viewport sizes will not be able to see all the navigation links within the expanded profile menu. Only the page _behind_ the expanded profile menu will scroll vertically leaving these users trapped. 

Since the profile menu is _sticky_ and placed _above_ the content of the page, users are still able to scroll the page itself, but not the contents of the sticky profile menu.

## Acceptance Criteria

- [ ] Users can view all navigation links within the expanded profile menu on zoom and smaller view port sizes

## Environment

* Operating System: any
* Browser: any at 400% zoom with a viewport of ~1180
* Screenreading device: any
* Server destination: staging

## Steps to Recreate

1. Enter `https://staging.va.gov/profile ` in browser
2. Zoom in to the page at 400% with a viewport of ~1180
3. Expand the sticky profile menu
4. Confirm you cannot scroll down to view the last link

## Proposed Solution

A possible solution would be to remove stickiness by deleting the css class `.the-menu.fixed`

## WCAG or Vendor Guidance (optional)

* [Gov.uk - Sticky elements functionality and accessibility testing](https://technology.blog.gov.uk/2018/05/21/sticky-elements-functionality-and-accessibility-testing/)

### Issues

https://user-images.githubusercontent.com/14154792/107566710-31937f00-6bb3-11eb-94fc-3cdd8f3d8127.mov

https://user-images.githubusercontent.com/14154792/107565419-b382a880-6bb1-11eb-8d5e-d2ca2a2328de.mov

###  Removing stickiness by removing `.the-menu.fixed` solves the issue

https://user-images.githubusercontent.com/14154792/107566848-5982e280-6bb3-11eb-9146-ed719cbeba75.mov

**Comment by Samara-Strauss** [comment]
This will likely be irrelevant when the nav is not sticky anymore.

**Comment by joshkimux** [comment]
@Samara-Strauss That's correct! Feel free to close this out once the nav is unstickied 😄

**Comment by Samara-Strauss** [comment]
Great! Sounds good :D We'll close this out once Erik has completed un-sticking the nav in #19779, which is slated for next sprint.

**Comment by erikphansen** [comment]
This issue does not exist anymore because the mobile nav is no longer "sticky."
